pkgs/development/tools/misc/ptags/default.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchFromGitHub
+, cargo
+, lib
+, rustPlatform
+, stdenv
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ptags";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "dalance";
+    repo = "ptags";
+    rev = "v${version}";
+    sha256 = "1xr1szh4dfrcmi6s6dj791k1ix2zbv75rqkqbyb1lmh5548kywkg";
+  };
+
+  cargoSha256 = "1rsnb4kzfb577xw7jk0939n42sv94vvspvbz783bmpy9vl53i38k";
+
+  # cargoBuildFlags = lib.optionals stdenv.isLinux [ "--target=x86_64-unknown-linux-musl" ]
+  #   ++ lib.optionals stdenv.isDarwin [ "--target=x86_64-apple-darwin" ];
+
+  # Failing for some reason...
+  doCheck = false;
```

Comment:
I started out trying this:

```diff
diff --git a/pkgs/development/tools/misc/ptags/default.nix b/pkgs/development/tools/misc/ptags/default.nix
index ff9f129f6a9..cd7fa02b9dc 100644
--- a/pkgs/development/tools/misc/ptags/default.nix
+++ b/pkgs/development/tools/misc/ptags/default.nix
@@ -3,6 +3,8 @@
 , lib
 , rustPlatform
 , stdenv
+, git
+, ctags
 }:
 
 rustPlatform.buildRustPackage rec {
@@ -14,6 +16,8 @@ rustPlatform.buildRustPackage rec {
     repo = "ptags";
     rev = "v${version}";
     sha256 = "1xr1szh4dfrcmi6s6dj791k1ix2zbv75rqkqbyb1lmh5548kywkg";
+    fetchSubmodules = true;
+    leaveDotGit = true;
   };
 
   cargoSha256 = "1rsnb4kzfb577xw7jk0939n42sv94vvspvbz783bmpy9vl53i38k";
@@ -21,8 +25,7 @@ rustPlatform.buildRustPackage rec {
   # cargoBuildFlags = lib.optionals stdenv.isLinux [ "--target=x86_64-unknown-linux-musl" ]
   #   ++ lib.optionals stdenv.isDarwin [ "--target=x86_64-apple-darwin" ];
 
-  # Failing for some reason...
-  doCheck = false;
+  buildInputs = [ ctags git ];
 
   meta = with stdenv.lib; {
     description = "A parallel universal-ctags wrapper for git repository";
```
but it doesn't seem to work.  

Ideally we'd get the whole test suite working, but if you want to just do `--help` it would be:

```suggestion
    checkPhase = ''
    $releaseDir/ptags --help
  '';
```
(Note that `$out/bin` does not yet exist.)

Reply:
Thanks :)